src/lib/snark_bits/tick_intf.ml

```diff
@@ -0,0 +1,133 @@
+(* tick_intf.ml -- enough of Snarky.Snark_intf.S to pass to Small_bit_vector functor
```

Comment:
Ditto with this filename, `Snarky_intf` it similar would be more representative.